src/lambda/explorer/containerNode.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import { TreeItem, TreeItemCollapsibleState } from 'vscode'
+import { AWSTreeNodeBase } from '../../shared/treeview/awsTreeNodeBase'
+import { NoFunctionsNode } from './noFunctionsNode'
+
+// Simple container node with a label
+export class ContainerNode extends AWSTreeNodeBase {
```

Comment:
This could be confused with container terminology, like Docker. Please rename to something the class/file to something like `GenericAwsTreeNode`